custom/ilsgateway/utils.py

```diff
@@ -64,3 +68,34 @@ def send_translated_message(user, message, **kwargs):
     with localize(user.get_language_code()):
         send_sms_to_verified_number(verified_number, message % kwargs)
         return True
+
+
+def make_loc(code, name, domain, type, metadata=None, parent=None):
```

Comment:
if these methods are only going to be used in tests I think it would be better to make a test utils module so as not to make it seem like they're used in live code